src/Native/Runtime/startup.cpp

```diff
@@ -214,13 +215,10 @@ void DllThreadDetach()
     }
 }
 
-void __stdcall FiberDetach(void* lpFlsData)
+void __stdcall ThreadShutdown()
```

Comment:
Nit:  this name is fairly generic.  And, given that this is a contract with the PAL, I wonder if we should have a less-generic name.  My suggestion would be `RuntimeThreadShutdown`, although I must admit that even this is fairly generic.